lib/_http_incoming.js

```diff
@@ -94,13 +94,13 @@ IncomingMessage.prototype._read = function(n) {
 // It's possible that the socket will be destroyed, and removed from
 // any messages, before ever calling this.  In that case, just skip
 // it, since something else is destroying this connection anyway.
-IncomingMessage.prototype.destroy = function(error) {
+IncomingMessage.prototype.destroy = function destroy(error) {
   if (this.socket)
     this.socket.destroy(error);
 };
 
 
-IncomingMessage.prototype._addHeaderLines = function(headers, n) {
+IncomingMessage.prototype._addHeaderLines = function _addHeaderLines(headers, n) {
```

Comment:
Please make sure the lines don't exceed 80 characters. `make jslint` should help you spot the long lines.